README.md

```diff
@@ -1,6 +1,16 @@
 # Goofi
 
-Let's find good first issues to contribute OSS. This is a simple script to list up issues labelled as "good first issue" in repositories which has over 500 stars.
+Let's find good first issues to contribute OSS. 
+
+This is a simple script to list issues labelled as "good first issue" in repositories with over 500 stars. This script lists the repository name, owner, and URL for each issue. The languages searched include the following:
+* JavaScript
+* TypeScript
+* Ruby
+* Go
+* Java
+* Python
+* Shell
```

Comment:
@htjjen 
Actually, this job doesn't limit language to the 7 above. A language can vary as long as GitHub can recognize the value.

You can find which language we can specify on GitHub trending page. Then you see there're quite many languages.

![image](https://user-images.githubusercontent.com/1811616/42011058-95776796-7acc-11e8-90b3-3d7b06fe72f8.png)

So my suggestion is, 

- Stop listing up languages
- Explain which language is available

What do you think?

Reply:
@ohbarye That makes sense! 
Perhaps I can change it to "Any language that can be recognized by GitHub can be searched." ?

Reply:
That sounds 🆒

Reply:
I've committed the change and sent the PR. Thank you!